charmcraft/commands/store/__init__.py

```diff
@@ -57,16 +57,17 @@ class WhoamiCommand(BaseCommand):
     help_msg = "returns your login information relevant to the store"
 
     _titles = [
-        ('name:', 'display-name'),
+        ('name:', 'name'),
         ('username:', 'username'),
-        ('id:', 'id'),
+        ('id:', 'userid'),
     ]
 
     def run(self, parsed_args):
         """Run the command."""
-        client = Client()
-        result = client.get('/v1/whoami')
+        store = Store()
+        result = store.whoami()
 
-        longest_title = max(len(t[0]) for t in self._titles)
-        for title, key in self._titles:
-            logger.info("%-*s %s", longest_title, title, result[key])
+        data = [(title, getattr(result, attr)) for title, attr in self._titles]
```

Comment:
why the indirection via `_titles`? I don't get what value it adds.

Reply:
indeed